internal/grpc/commander/commander.go

```diff
@@ -73,31 +72,10 @@ func (c *Commander) Download(_ *proto.DownloadRequest, _ proto.Commander_Downloa
 	return nil
 }
 
-// Upload is not needed by NKG but if we don't implement it the agent will spew error messages.
-// An active (connected) connection must exist in the AgentManager for the Upload to proceed.
-func (c *Commander) Upload(server proto.Commander_UploadServer) error {
+func (c *Commander) Upload(_ proto.Commander_UploadServer) error {
 	c.logger.Info("Commander Upload requested")
```

Comment:
do we need to log an error here in case we receive an unexpected upload request? 
would it make sense to return an error?

Reply:
I think we can leave the log as is, I don't think it's really an error, just an unexpected request.

It would make sense to return Unimplemented gRPC error to the agent though. I'll add that